test/cases/base_tests.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -euo pipefail
 
+#
+# Script that servers as a base for some tests
```

Comment:
What do you mean "base for some tests"? Look at line 16 this runs various high level and sanity tests including cli tests which are written in GO. Check also some logs from a run in Gitlab CI to see how it looks.